instrumentation/servlet/servlet-2.2/src/main/java/io/opentelemetry/javaagent/instrumentation/hypertrace/servlet/v2_2/nowrapping/Servlet2AndFilterInstrumentation.java

```diff
@@ -179,7 +178,6 @@ public static void exit(
             VirtualField.find(HttpServletRequest.class, StringMapSpanPair.class);
 
         // capture response body
-        // TODO: capture response headers
```

Comment:
Are we able to capture response headers?

Reply:
we are capturing `content-type` header but other response headers are not getting captured, at least in case of Boomi